zerver/openapi/zulip.yaml

```diff
@@ -1133,6 +1133,13 @@ paths:
                                 confusing users when someone comes online and immediately sends
                                 a message (one wouldn't want them to still appear offline at
                                 that point!).
+
+                                If the `CAN_ACCESS_ALL_USERS_GROUP_LIMITS_PRESENCE` server-level
+                                setting is set to `true`, then the event is only sent to users
+                                who can access the user who came back online.
+
+                                **Changes**: Prior to Zulip 8.0 (feature level 227), this event
+                                was sent to all users in the organization.
```

Comment:
I am not much sure about this changes, and might be phrased in a better way.

Reply:
This is probably OK but might deserve whatever editing pass we apply to the other discussion of accessing users.